HTTP2-Core.package/H2Session.class/instance/newH2WebResponseFor..st

```diff
@@ -0,0 +1,7 @@
+sending
+newH2WebResponseFor: aPromise
+	^ (H2WebResponse new 
+				extractDataH2Response: ((aPromise waitTimeoutSeconds: 3) ifTrue:[aPromise value]);
+				extractDataH2Session: self;
+				url: 'test.de.url'; 
```

Comment:
If `test.de.url` is supposed to be an invalid or testing DNS name, use `.invalid` or `.test` as per [RFC 2606](https://tools.ietf.org/html/rfc2606).